src/locale/locales/ca/messages.po

```diff
@@ -5724,7 +5724,7 @@ msgstr "Aquesta apel·lació s'enviarà a <0>{0}</0>."
 
 #: src/screens/Messages/Conversation/MessageListError.tsx:26
 msgid "This chat was disconnected due to a network error."
-msgstr ""
+msgstr Aquest xat s'ha desconnectat degut a un problema de xarxa.""
```

Comment:
There's a `"` missing at the beginning here and an extra `"` at the end.

Reply:
Thanks! your corrections are so useful!